src/Arcus.Security.Providers.AzureKeyVault/Authentication/Interfaces/IKeyVaultAuthenticator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.KeyVault;
+
+namespace Arcus.Security.Providers.AzureKeyVault.Authentication.Interfaces
+{
+    /// <summary>
+    ///     Authentication provider for Azure Key Vault
+    /// </summary>
+    [Obsolete(
+        "Use the "
```

Comment:
Why are we not just putting this in one line?

Reply:
It was otherwise very long line to read; and although I thought it would make it less readable to place it separatly; I liked it more this way.

Reply:
It was otherwise very long line to read; and although I thought it would make it less readable to place it separatly; I liked it more this way.

Reply:
I would move it all in one line as it creates a bit of noise but that's ok I guess.